tex/gregoriotex-nabc.lua

```diff
@@ -71,6 +71,15 @@ local function gregallreadfont(font_id)
   local tab = {}
   local metrics = {}
   local fontdata = fonts.hashes.identifiers[font_id]
```

Comment:
I think it would be good to use `font.getfont` instead of accessing `fonts.hashes.identifiers`, as suggested in [this commit](https://github.com/phi-gamma/gregorio/commit/392e77a0c3c80e6da2d633ba3cc560b03644fa2a)